include/cuco/bit_vector.cuh

```diff
@@ -59,51 +91,100 @@ class bit_vector {
   bit_vector();
   ~bit_vector();
 
-  void add(bool bit);  // adds a new bit at the end
-  void build();        // builds indexes for rank and select.
-
+  /**
+   * @brief adds a new bit at the end
+   *
+   * @param bit Boolean value of new bit to be added
+   */
+  void add(bool bit);
```

Comment:
I would name this function `append(bool bit)`

Reply:
see 6b553fc314e86c83589496dca0d19cf26df5b9c3